engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -644,6 +644,15 @@ protected void serviceStart() throws Exception
       if (SecurityUtils.isStramWebSecurityEnabled()) {
```

Comment:
It should not matter whether the function can be reused or not. It is an encapsulation of the logic and the functionality. StreamingAppMasterService.java is already over 13 hundred lines of code, so any logic that can be offloaded to another class should be moved there.